Sources/DockProgress/util.swift

```diff
@@ -41,7 +41,11 @@ final class ProgressCircleShapeLayer: CAShapeLayer {
 		self.init()
 		fillColor = nil
 		lineCap = .round
+		position = center
 		path = NSBezierPath.progressCircle(radius: radius, center: center).cgPath
+		if let boundingBox = path?.boundingBox {
+			bounds = boundingBox
+		}
```

Comment:
As far as I can remember, the `ProgressCircleShapeLayer` had no bounds or position values, this made it very hard to align the `CATextLayer` with it. They don't break anything else, they just add some extra correct values to it. 

I figured I'd assign these values so they are correct and can be used to center the `CATextLayer` and give it the same frame so that the dimensions match too. I had a really hard time getting the `CATextLayer` centered horizontally and vertically correct with the badge itself.

If you know of a better way to go about this, or don't agree with this I'd be happy to change it up.